.travis.yml

```diff
@@ -36,3 +36,16 @@ install:
 
 script:
   - python -m pytest
+
+env:
+  global:
+  # Doctr deploy key for Quansight/lsst_dashboard
+  - secure: "JZZB5EP/CYjRirqkW2vu+P+stLgwgqpTs68hSlto7uBDT7nRtQfElGCl/vwarr8Xlv2+iEJeGf3kjysZsuKX8wpQ7XDuKUopnlmSl+Esf6UVLY2zq3f9JBuEMl9BHlS/5e2lHs3QFdvSU40xD4SCTYY7bLU5M9Sd1FMS9mjV8SX7UN8F1mC+2HupZ8RWeZ02vj3LXjtZ2Zr0Hr3ouwMi/QY18ewTQralrVVCOZKzED6suhLJythoWO8Qw0R42YYUpFXxyV1ZKU27sf4QcsS9vbAbHYYr+8qGvGtyJf9R+8N2AqphPKJe5Pa/49MN/yjaZH0R26NaiHSdeqFZ43s9Ui2/I6y7Lheh5g4jJnsL5ORMIcjg4JCgRT+ACNswYas5tZEIG7CMfDruInM3alrdQWAAAnC7Lnc5jczH31dQmc+jFGqkgunl3RRP0ryNfe+pFeK38DI21L9Hgt5DQSuj9Bndhowsi/YfqvhhyXU8GmAZEUfsJuR9XY2oCkqv+N5QmMYrFzr4CepzYquWuFJOCujzDRBFRuV75l5DY3gT6InYHxnjR0HZHPl7byrsbQWQFZs/ZMKz+/nyjqLzdEOK277/z3sJCSMW+LAqnfNVdxWwWvnEFQk7xz4azjW8zmA7aIDS7Qi7LHto1Ho6tQ4OKqITHC4miNSQRTWocgeobSY="
```

Comment:
This looks dangerous. I think this key lets anyone push to the repo. Need to read up on how to set this up properly so it is secure.

Reply:
The following is the note that is included in the doctr documentation.  Does it seem like it will settle the problem to enable branch protection or does have that key out there at all seem too dangerous?

Note: That repo should already be set up with Travis. We recommend enabling branch protection for the gh-pages branch and other branches, as the deploy key used by Doctr has the ability to push to any branch in your repo.

Reply:
Also, I found this about security: 
Is this secure?

Doctr creates an encrypted SSH deploy key, which allows any Travis build on your repo to push to the deploy repo. The deploy key is encrypted using Fernet encryption from the Python cryptography module. The Fernet key is then encrypted to a secure environment variable for Travis using the Travis public key.

Travis does not make secure environment variables available to pull requests builds. Furthermore, Doctr itself does not push from any branch other than master by default, although this can be changed.

By default, Doctr uses deploy keys, but it can also use a GitHub personal access token, using the --token flag. However, this is not recommended, as a GitHub personal access token grants access to your entire account, whereas a deploy key only grants push access only to a single repository.

Both Doctr and Travis CI itself take measures to prevent the private encryption key from leaking in the build logs.

At any time, you can revoke the deploy key created by Doctr by going to the deploy key settings for the repository in GitHub at https://github.com/org/repo/settings/keys. Personal access tokens can be revoked at https://github.com/settings/tokens. If you revoke a key, you will need to rerun doctr configure to generate a new one to continue using Doctr.